cmd/zarb/main.go

```diff
@@ -21,9 +22,13 @@ func zarb() *cli.Cli {
 		k.Command("verify", "Verify a signature", key.Verify())
 		k.Command("change-auth", "Change the passphrase of a keyfile", key.ChangeAuth())
 	})
-	app.Command("tx", "Create raw transaction", func(k *cli.Cmd) {
-		k.Command("bond", "Generate raw bond transaction", tx.BondTx())
-		k.Command("send", "Generate raw send transaction", tx.SendTx())
+	app.Command("tx", "Create, Sing, Publish transactions", func(k *cli.Cmd) {
```

Comment:
`Create, sign and publish a transaction` is better. 
sign or sing ;)

Reply:
you right my bad